assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -1321,6 +1321,8 @@ public String buildMainPanelContext(VelocityPortlet portlet, Context context, Ru
         } catch (IdUnusedException iue) {
             log.warn(this + ":buildMainPanelContext: Site not found!" + iue.getMessage());
         }
+        User user = (User) state.getAttribute(STATE_USER);
+        context.put("isStudent", s.getUserRole(user.getId()).getId().equals("Student"));
```

Comment:
The role could be different than "Student" in project sites or other custom site types, we need a different approach

Reply:
Ok. I will see about something that could work